screengrab/example/fastlane/README.md

```diff
@@ -0,0 +1,49 @@
+fastlane documentation
```

Comment:
Woah, I don't remember documenting this...

Reply:
fastlane automatically creates the `README.md` when running lanes 😉